test/e2e/support/commands.js

```diff
@@ -133,6 +134,19 @@ Cypress.Commands.add("clearupDeployments", () => {
   cy.exec(`rm -rf content-workspace/static/.posit`);
 });
 
+Cypress.Commands.add("loadProjectConfig", (projectName) => {
```

Comment:
If we rename this a bit, we'll be ready for the deployment function... ;-)

```suggestion
Cypress.Commands.add("loadProjectConfigFile", (projectName) => {
```